src/pvt.c

```diff
@@ -271,8 +271,9 @@ static u8 filter_solution(gnss_solution* soln, dops_t* dops)
    * regulations. It must not be removed. Any modification to this condition
    * is strictly not approved by Swift Navigation Inc. */
 
-  if (vector_norm(3, soln->vel_ecef) >= 0.514444444*1000)
-    /* Velocity is greater than 1000kts. */
+  if ((vector_norm(3, soln->vel_ecef) >= 0.514444444*1000)
+      && (soln->pos_llh[2] > 18000))
+    /* COCOM limits: velocity greater than 1000 kts and altitude > 18 km. */
```

Comment:
Please revert this, COCOM does state altitude > 18km but DOC states just 1000kts (no minimum altitude).